src/dmd/astbase.d

```diff
@@ -3933,21 +3933,33 @@ struct ASTBase
 
     extern (C++) class TypeFunction : TypeNext
     {
-        ParameterList parameterList;  // function parameters
-
-        bool isnothrow;             // true: nothrow
-        bool isnogc;                // true: is @nogc
-        bool isproperty;            // can be called without parentheses
-        bool isref;                 // true: returns a reference
-        bool isreturn;              // true: 'this' is returned by ref
-        bool isscope;               // true: 'this' is scope
-        bool islive;                // true: function is @live
+        // .next is the return type
```

Comment:
This seems out of place?

Reply:
no it's just a hint coming from the real AST that explain with TypeClass decrivates from TypeNext